SConstruct

```diff
@@ -638,11 +638,8 @@ if env["nls"]:
     env.InstallData("datadir", "wesnoth", "l10n-track")
 InstallManpages(env, "wesnoth")
 if have_client_prereqs and have_X and env["desktop_entry"]:
-     if sys.platform == "darwin":
-         env.InstallData("icondir", "wesnoth", "icons/wesnoth-icon-Mac.png")
-     else:
-         env.InstallData("icondir", "wesnoth", "icons/wesnoth-icon.png")
-     env.InstallData("desktopdir", "wesnoth", "icons/wesnoth.desktop")
+     env.InstallData("icondir", "wesnoth", "packaging/icons")
+     env.InstallData("desktopdir", "wesnoth", "packaging/wesnoth.desktop")
```

Comment:
While cmake copies explicitly the hicolor folder, scons copies the contents of the icons folder, which ends up being the same.